gui/qt/__init__.py

```diff
@@ -194,25 +194,29 @@ def start_new_window(self, path, uri):
         else:
             try:
                 wallet = self.daemon.load_wallet(path, None)
+                if not wallet:
+                    storage = WalletStorage(path, manual_upgrades=True)
+                    wizard = InstallWizard(self.config, self.app, self.plugins, storage)
+                    try:
+                        wallet = wizard.run_and_get_wallet()
+                    except UserCancelled:
+                        pass
+                    except GoBack as e:
+                        print_error('[start_new_window] Exception caught (GoBack)', e)
+                    wizard.terminate()
+                    if not wallet:
+                        return
+                    wallet.start_threads(self.daemon.network)
+                    self.daemon.add_wallet(wallet)
             except BaseException as e:
                 traceback.print_exc(file=sys.stdout)
-                d = QMessageBox(QMessageBox.Warning, _('Error'), 'Cannot load wallet:\n' + str(e))
-                d.exec_()
+                if '2fa' in str(e):
+                    d = QMessageBox(QMessageBox.Warning, _('Error'), '2fa wallets are currently unsupported. Follow <a href="https://github.com/fyookball/electrum/issues/41#issuecomment-357468208">this guide</a> in order to recover your funds.')
```

Comment:
This makes it sound like our fault.  I think you should be explicit: "TrustedCoin, who hold the second signing key, do not support your 2FA wallet with Bitcoin Cash."

Reply:
How about "2FA wallets for Bitcoin Cash are currently unsupported by [TrustedCoin](https://api.trustedcoin.com/#/)"?

Reply:
Sure

Reply:
All fixed up!